blocks/cards/cards.js

```diff
@@ -27,6 +27,7 @@ export default function decorate(block) {
   /* change to ul, li */
   const ul = document.createElement('ul');
   [...block.children].forEach((row) => {
+    const linkName = row.querySelector('h5 a').getAttribute('title');
```

Comment:
should we better check in the two lines you added if the selector really returned something or are we 100% sure those elements always exist?

Reply:
The selector should return something because I did the tests and that's the section title. but anyway, I think you are right and a validation should exist anyway. I just added it.